scraper/github/queryManager.py

```diff
@@ -185,6 +185,7 @@ def queryGitHub(
         rest=False,
         requestCount=0,
         pageNum=0,
+        headers={},
```

Comment:
I think this (and `gitvars` and `keysToList` too actually) needs to be changed to default to `None` and then set the default (list or dict) in the code.

See: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Submitted fixes!